src/ebpf.rs

```diff
@@ -18,7 +18,7 @@ use byteorder::{ByteOrder, LittleEndian};
 use crate::stdlib::vec::Vec;
 
 /// Maximum number of instructions in an eBPF program.
-pub const PROG_MAX_INSNS: usize = 4096;
+pub const PROG_MAX_INSNS: usize = 1000000;
```

Comment:
This looks like it was picked by mistake while rebasing/merging, it's not related to the PR.

Reply:
Yes this was a merge, I will checkout the latest upstream and apply the clean changes there.